grafana-agent-operator.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: grafana-agent-operator
   version: 0.40.0
-  epoch: 0
+  epoch: 1
```

Comment:
```suggestion
  epoch: 0
```

Reply:
Test changes don't need epoch bumps

Reply:
gotcha, thanks for pointing that out, I was just cargo-cult bumping to be sure I got things to build 👍